PaddleAudio/examples/dcase2021_task1b/evaluate.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+import numpy as np
+import paddle
+import paddle.nn as nn
+import paddle.nn.functional as F
+import paddleaudio as pa
+import yaml
+from dataset import get_val_loader
+from model import resnet50
+from paddle.utils import download
+
+checkpoint_url = {
+    'audio_only':
+    'https://bj.bcebos.com/paddleaudio/paddleaudio/dcase21task1b_resnet50_audio_only_mixup=True_epoch188_acc0.771.pdparams',
```

Comment:
The bos folder structure seems not well planed.

Reply:
It would be better to distinguish model, dataset, temporary files etc.

Reply:
fixed.